sdk/wallet-adapter/adapters/unsafe-burner/src/index.ts

```diff
@@ -27,7 +27,7 @@ export class UnsafeBurnerWalletAdapter implements WalletAdapter {
   #signer: RawSigner;
   #account: ReadonlyWalletAccount;
 
-  constructor(network: Connection = devnetConnection) {
+  constructor(network: Connection = localnetConnection) {
```

Comment:
I changed this default since we expect more folks to do local network development.